llama_hub/microsoft_sharepoint/base.py

```diff
@@ -74,11 +82,12 @@ def _get_access_token(self) -> str:
             logger.error(response.json()["error"])
             raise ValueError(response.json()["error_description"])
 
-    def _get_site_id_with_host_name(self, access_token, sharepoint_site_name) -> str:
+    def _get_site_id_with_host_name(self, sharepoint_site_name: str) -> str:
         """
         Retrieves the site ID of a SharePoint site using the provided site name.
 
         Args:
+            access_token (str): access_token
```

Comment:
Since    `_get_site_id_with_host_name` does not have access_token as a argument. We can remove this from the docstring as well